src/components/Book.js

```diff
@@ -82,13 +90,14 @@ const Book = ({ book }) => {
       </div>
       <div className={css.BookActions}>
         <ActionButton title="Comments" />
+        <div className={css.LineSep} />
         <ActionButton trigger={onDelete} title="Remove" />
+        <div className={css.LineSep} />
         <ActionButton title="Edit" />
       </div>
```

Comment:
- [ ] Action buttons should be aligned to the left without padding:

Your design  | Zeplin design
------------- | -------------
<img width="245" alt="Screenshot 2022-07-15 at 16 40 04" src="https://user-images.githubusercontent.com/77038610/179224835-df528c29-0e24-4eeb-b369-c48677a6b711.png"> | <img width="347" alt="Screenshot 2022-07-15 at 16 41 17" src="https://user-images.githubusercontent.com/77038610/179224992-b05926e2-6fba-4fe8-a3b1-865ac05de742.png">